esp/bindings/SOAP/Platform/soapbind.hpp

```diff
@@ -220,6 +220,31 @@ class esp_http_decl CHttpSoapBinding : public CSoapBinding, public EspHttpBindin
     IEspContainer* queryContainer() { return m_container; }
     virtual int HandleSoapRequest(CHttpRequest* request, CHttpResponse* response);
 
+    virtual bool subscribeBindingToDali()
+    {
+        //can a binding have more than 1 service? if so, why is there a getService()?
```

Comment:
Yes, an ESPbinding only has 1 ESPservice.  getSerice() should return that one service.
Not to be confused with something like DynamicESDL that has virtual bindings and services.
Note: An ESP process can have multiple ports, each port can run multiple bindings.. each binding has one service but that can be many within a given process.

You should remove these comments once you are satisfied with the answer.